ddtrace/llmobs/_integrations/bedrock.py

```diff
@@ -87,6 +101,68 @@ def _llmobs_set_tags(
             }
         )
 
+    @staticmethod
+    def _extract_input_message_for_converse(prompt: List[Dict[str, Any]]) -> List[Dict[str, Any]]:
+        """Extract input messages from the stored prompt.
+        Anthropic allows for messages and multiple texts in a message, which requires some special casing.
+        """
+        if not isinstance(prompt, list):
+            log.warning("Bedrock input is not a list of messages or a string.")
+            return [{"content": ""}]
+        input_messages = []
+        for p in prompt:
+            if not isinstance(p, dict):
+                continue
+            role = str(p.get("role", ""))
+            content = p.get("content", "")
+            if isinstance(content, list):
+                if content and isinstance(content[0], dict):
+                    combined_text = ""
+                    for entry in content:
+                        if entry.get("text"):
+                            combined_text += entry.get("text", "") + " "
+                        else:
+                            content_type = ",".join(entry.keys())
+                            input_messages.append(
+                                {"content": "[Unsupported content type: {}]".format(content_type), "role": role}
+                            )
+                    if combined_text:
+                        input_messages.append({"content": combined_text.strip(), "role": role})
+
+        return input_messages
+
+    @staticmethod
+    def _extract_output_message_for_converse(response: Dict[str, Any]) -> List[Dict[str, Any]]:
+        # Handle Converse API response format with output field structure
+        if "output" in response and "message" in response.get("output", {}):
+            message = response.get("output", {}).get("message", {})
+            role = message.get("role", "assistant")
+            tool_calls_info = []
+            if message.get("content") and isinstance(message["content"], list):
+                content_blocks = []
+                for content_block in message["content"]:
+                    if content_block.get("text") is not None:
+                        content_blocks.append(content_block.get("text", ""))
+                    elif content_block.get("toolUse") is not None:
+                        tool_calls_info.append(
+                            {
+                                "name": content_block.get("toolUse", {}).get("name", ""),
+                                "arguments": content_block.get("toolUse", {}).get("input", ""),
+                                "tool_id": content_block.get("toolUse", {}).get("toolUseId", ""),
+                            }
+                        )
+                    else:
+                        log.debug(
+                            "LLM Obs tracing is unsupported for content block type: %s", ",".join(content_block.keys())
+                        )
```

Comment:
I think this warning debug log is unnecessary, we already have a visible content placeholder anyway